data/countries-bulk.json

```diff
@@ -0,0 +1,398 @@
+{"index" : {}}
```

Comment:
Should we commit these, rather than just generating them temporarily when creating the index?

Reply:
Yeh, still in progress really